Updating self-service page (#6221)
dbt-labs/docs.getdbt.com: website/docs/docs/cloud/manage-access/self-service-permissions.md

---
title: "Self-service Team account permissions"
description: "Learn how dbt Cloud administrators can use self-service permissions to control access in a dbt Cloud Team account."
sidebar_label: "Team permissions"
id: "self-service-permissions"
---

> Self-service Team accounts are a quick and easy way to get dbt Cloud up and running for a small team. Team accounts have limited access to features and advanced settings like SSO, group management, and larger user bases will require an [Enterprise](/docs/cloud/manage-access/enterprise-permissions) account. If you're interested in upgrading, contact [dbt Labs today](https://www.getdbt.com/contact)!

Reviewer: maybe a bit more positive
```suggestion
Self-service Team accounts are a quick and easy way to get dbt Cloud up and running for a small team. For teams looking to scale and access advanced features like SSO, group management, and support for larger user bases, upgrading to an [Enterprise](/docs/cloud/manage-access/enterprise-permissions) account unlocks these capabilities. 

If you're interested in upgrading, contact [dbt Labs today](https://www.getdbt.com/contact)
```

## Groups and permissions

> Groups determine a users permission and there are three groups are available for Team plan dbt Cloud accounts: Owner, Member, and Everyone. The first Owner user is the person who created the dbt Cloud account. New users are added to the Member and Everyone groups when they onboardbut this can be changed when the invitation is created. These groups only affect users with a [Developer license](#licenses) assigned. 

Reviewer: ```suggestion
Groups determine a user's permission and there are three groups are available for Team plan dbt Cloud accounts: Owner, Member, and Everyone. The first Owner user is the person who created the dbt Cloud account. 

New users are added to the Member and Everyone groups when they onboard but this can be changed when the invitation is created. These groups only affect users with a [Developer license](#licenses) assigned. 
```

> Groups determine a users permission and there are three groups are available for Team plan dbt Cloud accounts: Owner, Member, and Everyone. The first Owner user is the person who created the dbt Cloud account. New users are added to the Member and Everyone groups when they onboardbut this can be changed when the invitation is created. These groups only affect users with a [Developer license](#licenses) assigned. 

Reviewer: can you delete any of these?

The group access permissions are as follows:

- **Owner** &mdash; Full access to account features.
- **Member** &mdash; Robust access to the account with restrictions on features that can alter billing or security.
- **Everyone** &mdash; A catch-all group for all users in the account. This group does not have any permission assignments beyond the user's profile. Users must be assigned to either the Member or Owner group to work in dbt Cloud. 

## Licenses

You assign licenses to every user onboarded into dbt Cloud. You only assign Developer-licensed users to the Owner and Member groups. The groups have no impact on Read-only or IT licensed users.

There are three license types:

- **Developer** &mdash; The default license. Developer licenses don't restrict access to any features, so users with this license should be assigned to either the Owner or Member group. You're allotted up to 8 developer licenses per account.
- **Read-Only** &mdash; Read-only access to your project, including environments dbt Explorer. Doesn't have access to account settings at all. Functions the same regardless of group assignments. You're allotted up to 5 read-only licenses per account.

> - **IT** &mdash; Partial access to the account settings including users, integrations, billing, and API settings. Cannot create or edit connects or access the project at all. Functions the same regardless of group assignments.

Reviewer: following the earlier patterns -- should we state how many IT licenses users are allowed?
```suggestion
- **IT** &mdash; Partial access to the account settings including users, integrations, billing, and API settings. Cannot create or edit connects or access the project at all. Functions the same regardless of group assignments. You're allocated 1 seat per account.
```

See [Seats and Users](/docs/cloud/manage-access/seats-and-users) for more information on the impact of licenses on these permissions.

## Table of groups, licenses, and permissions

> 

Reviewer: For this whole section, I wonder if it's maybe a bit simpler to:
- have an intro sentence before the key
- then repurpose the permissions bullets for each table so there's an intro section before each table
- make it clear in the first column that the fields are for UI fields in dbt Cloud (i struggled at first)

just a thought!

e.g.:
```
## Table of groups, licenses, and permissions

The following tables outline the access that users have if they are assigned a Developer license and the Owner or Member group, Read-only license, or IT license. Each role grants different levels of access to specific user interface (UI) fields within the Cloud UI, such as Connections, Credentials, and more.

Key:
(W)rite — Create new or modify existing. Includes send, create, delete, allocate, modify, and read.
(R)ead — Can view but can not create or change any fields.
No value — No access to the feature.

#### Account-level permissions
These permissions apply to UI fields that manage the overall dbt Cloud account, such as account settings, billing, and user invitations.
| UI field | Owner | Member | Read-online license | IT license |
| ------- | ------ | -------- | ------------------- | ---------- |
...text. 

```

Author: There is a project in October to completely overhaul this whole permissions tables thing. Will take it up with that project.

Key:

* (W)rite &mdash; Create new or modify existing. Includes `send`, `create`, `delete`, `allocate`, `modify`, and `read`.
* (R)ead &mdash; Can view but can not create or change any fields.
* No value &mdash; No access to the feature.

Permissions:

> * Account-level permissions &mdash; Permissions related to management of the dbt Cloud account. For example, billing and account settings.

Reviewer: ```suggestion
* [Account-level permissions](#account-permissions-for-account-roles) &mdash; Permissions related to management of the dbt Cloud account. For example, billing and account settings.
```

> * Project-level permissions &mdash; Permissions related to the projects in dbt Cloud. For example, Explorer and the IDE.

Reviewer: ```suggestion
* [Project-level permissions](#project-permissions-for-account-roles) &mdash; Permissions related to the projects in dbt Cloud. For example, Explorer and the IDE.
```

| Account settings        |   W   |   W    |                  |       W      |
| Billing                 |   W   |        |                  |       W      |
| Invitations             |   W   |   W    |                  |       W      |
| Licenses                |   W   |   R    |                  |       W      |
| Users                   |   W   |   R    |                  |       W      |
| Project (create)        |   W   |   W    |                  |       W      |
| Connections             |   W   |   W    |                  |       W      |
| Service tokens          |   W   |        |                  |       W      |
| Webhooks                |   W   |   W    |                  |              |
 
#### Project permissions for account roles

|Project-level permission | Owner | Member  | Read-only | IT license |
|:------------------------|:-----:|:-------:|:---------:|:----------:|
| Adapters                |   W   |    W    |    R      |             |
| Connections             |   W   |    W    |    R      |             |
| Credentials             |   W   |    W    |    R      |             |
| Custom env. variables   |   W   |    W    |    R      |             |

> | dbt adapters            |   W   |    W    |           |             |

Reviewer: what does dbt adapters mea? how is it diff to connections?

Author: Good catch. Removing that

| Develop (IDE or dbt Cloud CLI)| W |  W    |           |             |
| Environments            |   W   |    W    |    R      |             |
| Jobs                    |   W   |    W    |    R      |             |

> | Metadata                |   R   |    R    |    R      |             |

Reviewer: by metadata, do we mean explorer?

Author: I'm not sure. This was already in the table when I moved it. It's all read-only so I don't think it's Explorer as a whole, but I'll remove it for now and revisit when we update these tables

| Licenses                |   W   |   R    |                  |       W      |
| Users                   |   W   |   R    |                  |       W      |
| Project (create)        |   W   |   W    |                  |       W      |
| Connections             |   W   |   W    |                  |       W      |
| Service tokens          |   W   |        |                  |       W      |
| Webhooks                |   W   |   W    |                  |              |
 
#### Project permissions for account roles

|Project-level permission | Owner | Member  | Read-only | IT license |
|:------------------------|:-----:|:-------:|:---------:|:----------:|
| Adapters                |   W   |    W    |    R      |             |
| Connections             |   W   |    W    |    R      |             |
| Credentials             |   W   |    W    |    R      |             |
| Custom env. variables   |   W   |    W    |    R      |             |
| Develop (IDE or dbt Cloud CLI)| W |  W    |           |             |
| Environments            |   W   |    W    |    R      |             |
| Jobs                    |   W   |    W    |    R      |             |

> | Metadata                |   R   |    R    |    R      |             |

Author: ```suggestion
| Explorer               |   W   |    W   |    R      |             |
```

> | Explorer               |   W   |    W   |    R      |             |

Author: ```suggestion
| dbt Explorer            |   W   |    W   |    R      |             |
```

| Account settings        |   W   |   W    |                  |       W      |
| Billing                 |   W   |        |                  |       W      |
| Invitations             |   W   |   W    |                  |       W      |
| Licenses                |   W   |   R    |                  |       W      |
| Users                   |   W   |   R    |                  |       W      |
| Project (create)        |   W   |   W    |                  |       W      |
| Connections             |   W   |   W    |                  |       W      |
| Service tokens          |   W   |        |                  |       W      |
| Webhooks                |   W   |   W    |                  |              |
 
#### Project permissions for account roles

|Project-level permission | Owner | Member  | Read-only | IT license |
|:------------------------|:-----:|:-------:|:---------:|:----------:|
| Adapters                |   W   |    W    |    R      |             |
| Connections             |   W   |    W    |    R      |             |
| Credentials             |   W   |    W    |    R      |             |
| Custom env. variables   |   W   |    W    |    R      |             |

> | dbt adapters            |   W   |    W    |           |             |

Author: ```suggestion
```
